src/libs/ajax.js

```diff
@@ -170,6 +171,31 @@ const User = signal => ({
       `${await Config.getTosUrlRoot()}/user/response`,
       _.mergeAll([authOpts(), { signal, method: 'POST' }, jsonBody({ ...tosData, accepted: true })])
     )
+  },
+
+  createSupportRequest: async ({ name, email, currUrl, subject, type, description }) => {
+    const options = {
+      method: 'POST',
+      body: JSON.stringify({
+        request: {
+          requester: { name, email },
+          subject,
+          'custom_fields': [
+            { id: 360012744452, value: type },
+            { id: 360007369412, value: description },
+            { id: 360012744292, value: name },
+            { id: 360012782111, value: email }
+          ],
+          comment: {
+            body: `${description}\n\n------------------\nSubmitted from: ${currUrl}`
+          }
+        }
+      }),
+      headers: {
+        'Content-Type': 'application/json'
+      }
+    }
+    return await fetchOk(`https://broadinstitute.zendesk.com/api/v2/requests.json`, options)
```

Comment:
nit: `await` is unnecessary here, since the `async` function will return a promise anyway. This could be `return fetchOk(...)`